### ISSUE TITLE
Player statistics

### ISSUE THREAD

**Issue by herzbube** [herzbube/littlego#23]
There have been classes for player statistics since day zero, including support for storing the data. Now we only have to use them and display the values...

**Comment by theophanemayaud** [comment]
I agree this would be great ! It seemed weird when starting to use the app that there is a logic for creating players... but no actual use of them !

Would you be ok if I started working on it also next month ? I could start by simply saving statistics after each game ends, and displaying an average personal score for each player but also when clicking one player, displaying its scores against typical (most played against) opponents ?

**Comment by herzbube** [comment]
Feel free to record whatever you think is nice. As mentioned in the description there is already a ˋ PlayerStatisticsˋ class that you could use when you start working.
